spec/jobs/process_photo_job_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe ProcessPhotoJob, :type => :job do
+  subject { ProcessPhotoJob.new }
+
+  describe "#perform" do
+    let(:blob_storage) { double(upload: true) }
+    let(:image_path) { File.join(Rails.root, 'spec/fixtures/images/gps.jpg') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.